source/includes/usage-examples/time-series.rb

```diff
@@ -0,0 +1,36 @@
+require 'bundler/inline'
+gemfile do
+  source 'https://rubygems.org'
+  gem 'mongo'
+end
+
+Mongo::Client.new(uri) do |client|
+
+  # start-create
+  client = Mongo::Client.new(['<connection string>'], database: 'weather')
+  collection_name = 'october2024'
+
+  time_series_options = { timeField: 'timestamp' }
+  database = client.database
+  database.command(
+  create: collection_name,
+  timeseries: time_series_options
+  )
+  # end-create
+
+  # start-correct
+  collections = database.list_collections(filter: { name: 'october2024' }).to_a
+  puts collections
+  # end-correct
+
+  # start-insert
+  client = Mongo::Client.new(['<connection string>'], :database => 'your_db')
```

Comment:
As mentioned above, this should be either `[ '<uri>' ]`, or `'<connection string>'` (i.e. don't put connection strings in an array).

Also, `database: 'your_db'` is more idiomatic than `:database => 'your_db'`.